slf4j/src/main/scala/org/typelevel/log4cats/slf4j/LoggerName.scala

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright 2018 Typelevel
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.typelevel.log4cats.slf4j
+
+object LoggerName extends LoggerNamePlatform
+
+final case class LoggerName(value: String)
```

Comment:
I'm not a real fan of `AnyVal`, but here it seems actually will serve well. What do you think? Or maybe we could just add a type alias for the `String` and use it when needed?

Reply:
Type alias won't work because it is going to be the same as `implicit loggerName: String`, and nobody likes implicit strings

Reply:
Added the AnyVal suggestion